### DISCUSSION TITLE
Can Category Selection dropdown support search function?

### DISCUSSION THREAD

**Discussion by finlabhk** [fomantic/Fomantic-UI#2187] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hi, I would like to add the multiple search function on existing category selection dropdown. The function seems work but the options of sub group cannot be shown completely. Please refer to the link for reference.

[https://jsfiddle.net/0ea7mj2z/1/](url)

**Comment by lubber-de** [comment, marked as answer]
- Multiple search selection does not support submenus nor submenu entries.
- A search dropdown is supposed to be used on large dropdown lists, that's why there is `max-height` set to such menus by default. You can adjust the height by either [adding `long` or `very long`  to the dropdown](https://jsfiddle.net/lubber/3qktcme9/16/) or [override the max-height setting manually](https://jsfiddle.net/lubber/3qktcme9/12/)  
- A (multiple) search dropdown is supposed to be used in combination with[ `selection`](http://localhost:999/modules/dropdown.html#search-selection) only
